src/main/java/nl/ordina/jobcrawler/scrapers/VacancyScraper.java

```diff
@@ -1,97 +1,63 @@
 package nl.ordina.jobcrawler.scrapers;
 
-import lombok.extern.slf4j.Slf4j;
 import nl.ordina.jobcrawler.model.Vacancy;
-import nl.ordina.jobcrawler.model.VacancyURLs;
 import org.jsoup.Jsoup;
 import org.jsoup.nodes.Document;
 
 import java.io.IOException;
-import java.util.ArrayList;
-import java.util.HashSet;
 import java.util.List;
 
-// All vacancy scrapers have the following functions in common
-
-@Slf4j
-abstract class VacancyScraper {
-
-    private static final String userAgent = "Mozilla/5.0 (Windows NT 6.1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/41.0.2228.0 Safari/537.36 ArabotScraper";
+abstract public class VacancyScraper {
 
     private final String SEARCH_URL;
     private final String BROKER;
 
+    /**
+     * Constructor for abstract class VacancyScraper
+     *
+     * @param url Default seach url for scraper
+     * @param broker Used broker for scraper
+     */
     public VacancyScraper(String url, String broker) {
         this.SEARCH_URL = url;
         this.BROKER = broker;
     }
 
-    public static Document getDocument(final String url) throws IOException {
-        return Jsoup.connect(url).userAgent(userAgent).get();
+    /**
+     * The getDocument function retrieves the html page from the given url
+     *
+     * @param url Url from page that needs to be retrieved
+     * @return Jsoup Document
+     */
+    public Document getDocument(final String url) {
```

Comment:
Why not static?

Reply:
There is currently no need to have this method as static method.

Reply:
I think static means that only one per class gets made. so even if there are more instantiations of the VacancyScraper, only 1 method would be made. This works great for utility methods like these that do not need to access attributes. 
But indeed there are no instantiations of the class so it would be the same, I think.

Reply:
Might be an idea to discuss this request tomorrow? As tests for the scrapers might need to change a bit due to changes in this merge request?

Reply:
please make it static if possible, makes it clearer.